functions/Add-Noun.ps1

```diff
@@ -0,0 +1,39 @@
+function Add-Noun {
+    <#
+        .SYNOPSIS
+            Adds a Noun 
+
+        .DESCRIPTION
+            Longer description of what Add-Noun does
+
+        .PARAMETER ComputerName
```

Comment:
Do we also want to add the `SqlInstance` to this just so they can remove the one not needed?

It will be based on what action the command is doing as to whether they use one over the other.